_timeline/2017-11-02-prototype-release-0.1.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Prototype release v0.1
+date: 2017-11-02
+---
+Prototype script [v0.1](https://github.com/tomeshnet/prototype-cjdns-pi/releases) was released. It now supports some Armbian boards as well as the newest Raspbian.
```

Comment:
Script for what? "Prototype mesh node install script" (or something... just more info!)

Reply:
Also would say "Released [mesh node install script v0.1](), which supports the newest Raspbian and some Arbian boards.